### ISSUE TITLE
Fail headings elements at the beginning of each pie

### ISSUE THREAD

**Issue by MewenLeHo** [Orange-OpenSource/a11y-guidelines#688]
In our accessibility statement after disabling CSS, the content is placed before respective heading failing [Technique H69: Providing heading elements at the beginning of each section of content](https://www.w3.org/WAI/WCAG21/Techniques/html/H69)

![Image](https://github.com/user-attachments/assets/282a9bdb-fe1c-41e9-9cbb-26dcaac83fac)
